features/step_definitions/helper_steps.rb

```diff
@@ -67,8 +66,14 @@
 # Checks the mortgage deed is unsigned for borrower, then signs it.
 Given(/^the deed is digitally signed by borrower <(\d+)>$/) do |borrower|
   step %(the borrower <#{borrower}> signature element is present on page)
-  step %(I request an authentication code for borrower <#{borrower}>)
-  step %(I enter an authentication code)
+
```

Comment:
As discussed if the above passes then the authentication code text should always be there so would be better to put the conditional around the above step.